Update README.md

mizukosy opened the pull request:
what
READMEの作成
why
データベースを作成するための準備のため

mizukosy committed:
Update README.md

mizukosy commented:
ER図
https://gyazo.com/6fcdd4aa4153e16dae8ff6b7f512c03f

isyt729 commented on README.md:
```diff
@@ -22,3 +22,90 @@ Things you may want to cover:
+| ---------------- | ------ | ----------- |
+| nickname         | string | null: false |
+| password         | string | null: false |
+| password_check   | string | null: false |
```
| password_check   | string | null: false |
passwordは1カラムで十分ですので、削除しましょう！

mizukosy replied:
- [x password check 削除]

isyt729 commented on README.md:
```diff
@@ -22,3 +22,90 @@ Things you may want to cover:
+| Column               | Type       | Options                       |
+| -------------------- | ---------- | ----------------------------- |
+| item_name            | string     | null: false                   |
+| item_image           | string     | null: false                   |
```
| item_image           | string     | null: false                   |

画像はactive_storageで管理するため、不要です！

mizukosy replied:
- [x item_image削除]

isyt729 commented on README.md:
```diff
@@ -22,3 +22,90 @@ Things you may want to cover:
+| item_name            | string     | null: false                   |
+| item_image           | string     | null: false                   |
+| item_text            | text       | null: false                   |
+| item_category        | integer    | null: false                   |
+| item_status          | integer    | null: false                   |
+| item_price           | integer    | null: false                   |
+| item_price_fee       | integer    |                               |
+| item_delivery_chager | integer    | null: false                   |
+| item_delivery_area   | integer    | null: false                   |
+| item_delivery_days   | integer    | null: false                   |
```
| item_category        | integer    | null: false                   |
| item_status          | integer    | null: false                   |
| item_price_fee       | integer    |                               |
| item_delivery_chager | integer    | null: false                   |
| item_delivery_area   | integer    | null: false                   |
| item_delivery_days   | integer    | null: false                   |


active_hashを用いるカラムは_idを末尾につけましょう！

isyt729 commented on README.md:
```diff
@@ -22,3 +22,90 @@ Things you may want to cover:
+| ------ | ---------- | ----------------------------- |
+| user   | references | null: false, forgen_key: true |
+| item   | references | null: false, forgen_key: true |
+<!-- referenseで作るかどうかを検討 -->
```
不要なコメントは削除しましょう！

mizukosy replied:
- [x 不要なコメント削除 ]　失礼いたしました

isyt729 reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認お願いします！

mizukosy committed:
READMEの修正

password check,の削除
item_imageの削除
active_hashの部位(item_category, item_status, item_delivery_charger, item_delively_area, item_delivery_days)を修正

shingoteshima commented on README.md:
```diff
@@ -22,3 +22,87 @@ Things you may want to cover:
+
+- has_many :items
+- has_many :coments
+- has_one  :purchase
```
1人のuserは複数の商品を購入できるため、has_many :purchasesとしましょう。

mizukosy replied:
- [ x アソシエーションの修正]

shingoteshima commented on README.md:
```diff
@@ -22,3 +22,87 @@ Things you may want to cover:
+
+| Column                  | Type       | Options                       |
+| ----------------------- | ---------- | ----------------------------- |
+| item_name               | string     | null: false                   |
```
itemsテーブルのカラムなので、カラム名に先頭の`item_`は不要です。
他のカラムも同様です。

mizukosy replied:
- [ x itemの削除 ]

shingoteshima reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認お願いいたします！

mizukosy committed:
READMEの修正

itemテーブルの名前修正
usersとpurchasesのアソシエーションの修正

mizukosy committed:
READMEの修正

isyt729 commented:
レビュー依頼ありがとうございます！
LGTMといたします！
引き続き実装頑張っていきましょう！

mizukosy commented:
ありがとうございます。
マージいたします
